sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -52,7 +52,8 @@ async def test_async_queue_by_queue_client_conn_str_receive_handler_peeklock(sel
             async with sb_client.get_queue_sender(servicebus_queue.name) as sender:
                 for i in range(10):
                     message = Message("Handler message no. {}".format(i))
-                    await sender.send_messages(message)
+                    await sender.send_messages(message, timeout=5)
+                    assert 4990 <= sender._handler._msg_timeout <= 5000  # initial retry logic consumes cycles, usually just 1ms
```

Comment:
I'm a little nervous about this.  (in terms of flakiness.) since it's basically asking "did it succeed within 10ms" (did nothing go wrong; not even one retry.  Thoughts?

Reply:
sorry about this confusing you, it's not about op succeed or not, it's just when the program comes to set the timeout of sender._handler, the program has executed for several milliseconds, the remaining time would not be exactly the 5000.

I've removed this line as we have added timeout test to prove timeout is set and working as expected.